src/newcomponents/context/elements/LeaveAndDeleteLink.js

```diff
@@ -0,0 +1,17 @@
+import ContextLink from "@newcomponents/context/elements/ContextLink";
+import conversationService from "@services/conversationsService";
+
+import { ReactComponent as Leave } from "@icons/context/Leave.svg";
+
+export default function LeaveAndDeleteLink() {
+  const onClickFunc = async () => await conversationService.sendDeleteRequest();
```

Comment:
I propose to do this component less aware of outside world, and pass `onClickFunc ` via prop.

Also onClickFunc -> onClick